### PR TITLE
Replace references to CurationConcerns

### LINKED ISSUE #29: Replace vestiges of Sufia and CC from codebase
```diff
README.md
43:Hyrax uses the full power of [Hydra](http://projecthydra.org/) and extends it to provide a user interface around common repository features and social features (see below). Hyrax offers self-deposit and proxy deposit workflows, and mediated deposit workflows are being developed in a community sprint running from September-December 2016. Hyrax delivers its rich and growing set of features via a modern, responsive user interface. It is implemented as a Rails engine, so it is meant to be added to existing Rails apps. Hyrax is the consolidation of Sufia and the CurationConcerns gems and behaves in much the same way.
89:See the [Sufia Management Guide](https://github.com/projecthydra/sufia/wiki/Sufia-Management-Guide) to learn which features listed above are turned on by default and which require configuration.
91:For non-technical documentation about Hyrax, see the Sufia [documentation site](http://sufia.io/).
203:The [Sufia Management Guide](https://github.com/projecthydra/sufia/wiki/Sufia-Management-Guide) provides tips for how to manage, customize, and enhance your Hyrax application, including guidance specific to:
228:The [Sufia Development Guide](https://github.com/projecthydra/sufia/wiki/Sufia-Development-Guide) is for people who want to modify Hyrax itself, not an application that uses Hyrax.
232:See the [release management process](https://github.com/projecthydra/sufia/wiki/Release-management-process).
239:![Project Hydra Logo](http://sufia.io/assets/images/hydra_logo.png)
```

And find CC references via `grep -ri curationconcerns .`.

### PR DESCRIPTION
**Pull request by mjgiarlo**
Fixes #29

### CODE REVIEW

File: lib/generators/hyrax/templates/app/controllers/curation_concerns/admin_controller.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module CurationConcerns
+module Hyrax
```
Comment (jcoyne) [inline lib/generators/hyrax/templates/app/controllers/curation_concerns/admin_controller.rb]: I'm not sure we need this file. If we do, it should not have `curation_concerns` in its path.

Comment (mjgiarlo) [inline lib/generators/hyrax/templates/app/controllers/curation_concerns/admin_controller.rb]: @jcoyne OK. I'll get back to this next week, probably.

Comment (mjgiarlo) [inline lib/generators/hyrax/templates/app/controllers/curation_concerns/admin_controller.rb]: Got back to this and fixed the path. Not going to bother nixing this file in this PR.

Comment (mjgiarlo) [inline lib/generators/hyrax/templates/app/controllers/curation_concerns/admin_controller.rb]: @jcoyne re-review, pls?

**Comment by mjgiarlo** [comment]
After merging this PR, it may be time for another round of merges from sufia master and CC master.

File: lib/generators/hyrax/templates/config/hyrax.rb [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Hyrax.config do |config|
+Hyrax.configure do |config|
```
Comment (jcoyne) [inline lib/generators/hyrax/templates/config/hyrax.rb]: This method doesn't exist. Please revert to `config`

Comment (mjgiarlo) [inline lib/generators/hyrax/templates/config/hyrax.rb]: Just discovered that. Incoming.

**Pull request merged**